doc/classes/BackBufferCopy.xml

```diff
@@ -31,7 +31,7 @@
 			<argument index="0" name="copy_mode" type="int" enum="BackBufferCopy.CopyMode">
 			</argument>
 			<description>
-				Set the copy mode of the BackBufferCopy (refer to constants section).
+				Set the copy mode of the BackBufferCopy (see constants).
```

Comment:
To be consistent with the writing guide and the rest of the docs, you should write it as:

    See [code]COPY_MODE_*[/code] constants.